tee-worker/local-setup/worker_config_dev.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I don't see a difference between dev/staging/prod, this needs a clear definition: what exactly should differ in these different configs? -- needs to be evaluated/defined by who opens this issue (maybe @jingleizhang)